### ISSUE TITLE
Code injection in Twig

### ISSUE THREAD

**Issue by miguelvaara** [NatLibFi/Skosmos#1271]
## What steps will reproduce the problem?

Check these out:
1. https://github.com/NatLibFi/Skosmos/security/dependabot/1
2. Related ticket in the ticketing system

**Comment by miguelvaara** [comment]
Ref (https://github.com/NatLibFi/Skosmos/security/dependabot/1): _When in a sandbox mode, the arrow parameter of the sort filter must be a closure to avoid attackers being able to run arbitrary PHP functions_

**Comment by osma** [comment]
It's pretty hard to see how this could affect Skosmos. For one, we don't use the `sort` filter in the Twig templates:

```
git/Skosmos/view$ grep sort *
headerbar.twig:  {% if sorted_vocabs is defined %}
headerbar.twig:    {% for vocab in sorted_vocabs %}
scripts.twig:var sortByNotation = {% if request.vocab.config.sortByNotation %}"{{ request.vocab.config.sortByNotation }}"{% else %}null{% endif %};
```

There are very few cases where user provided input is used in the Skosmos UI. The only case I can think of is the search string, which is echoed on the search result page.

**Comment by osma** [comment]
We are currently using Twig 2.13: https://github.com/NatLibFi/Skosmos/blob/52e707930df8ab2bb2e8993cc7181e384cfa8163/composer.json#L64

Upgrading to Twig 2.14.11+ would fix the issue according to dependabot.

**Comment by joelit** [comment]
This was fixed in #1262 as it updates Twig
